Makefile

```diff
@@ -0,0 +1,21 @@
+SHELL=/bin/bash
+
+CC=gcc
+CFLAGS=-ggdb -Isrc -Istorj -luv -lnettle -lm -ljson-c
+
+COMMON_SRC=src/crypto.c src/utils.c src/storj.c src/uploader.c src/downloader.c storj/uplink.so
+COMMON_DEPS=src/storj.h storj/uplink.h storj/uplink.so
+
+CLI_DEPS=src/cli_callback.h $(COMMON_DEPS)
+CLI_SRC=src/cli.c $(COMMON_SRC)
+cli: libuplink $(CLI_SRC)
```

Comment:
I think each command in the Makefile should have a comment briefly explaining its functionality

Reply:
:+1: